lib/plausible/stats/imported.ex

```diff
@@ -28,28 +28,33 @@ defmodule Plausible.Stats.Imported do
     from(s in Ecto.Query.subquery(native_q),
       full_join: i in subquery(imported_q),
       on: field(s, :date) == field(i, :date),
-      select: %{date: field(s, :date)}
+      select: %{date: fragment("greatest(?, ?)", field(s, :date), field(i, :date))}
```

Comment:
this quickfix helps avoid zeroed dates on the main graph

more info: https://gist.github.com/ruslandoga/7940b1d9242fb0fefdf6530dcb212ccc